v2/pgdeploy

```diff
@@ -142,9 +142,9 @@ class PgBouncerUpdater(object):
 
                 # Reload pgbouncer
                 if remote_host["username"] == "root":
-                    shell.run(["service", "pgbouncer", "reload"])
+                    shell.run(["systemctl", "pgbouncer", "reload"])
                 else:
-                    shell.run(["sudo", "service", "pgbouncer", "reload"])
+                    shell.run(["sudo", "systemctl", "pgbouncer", "reload"])
```

Comment:
@mfuhrmann it would have to be `systemctl reload pgbouncer` though :P

Reply:
argh...